add option to disable compiler-rt scudo standalone allocator as it is
not always supported by all arch profiles such as arm5vt of ARM32 arch
Index: llvm-toolchain-snapshot_20~++20240920082629+ea578804c81b/compiler-rt/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_20~++20240920082629+ea578804c81b.orig/compiler-rt/CMakeLists.txt
+++ llvm-toolchain-snapshot_20~++20240920082629+ea578804c81b/compiler-rt/CMakeLists.txt
@@ -288,6 +288,12 @@ option(COMPILER_RT_USE_ATOMIC_LIBRARY "U
 
 include(config-ix)
 
+option(COMPILER_RT_BUILD_SCUDO_STANDALONE
+  "override whether or not to build scudo standalone" ON)
+if(NOT COMPILER_RT_BUILD_SCUDO_STANDALONE)
+  set(COMPILER_RT_HAS_SCUDO_STANDALONE FALSE)
+endif()
+
 #================================
 # Setup Compiler Flags
 #================================
